### PR TITLE
Multilang: add Portuguese (Brazil) translation

### PR DESCRIPTION
**Pull request by AlexandreSato**

### CODE REVIEW

File: selfdrive/ui/translations/main_pt.ts [hunk 1]
```diff
@@ -0,0 +1,1376 @@
...
+        <translation>dashcam</translation>
+    </message>
+    <message>
+        <location filename="../qt/util.cc" line="21"/>
+        <source>openpilot</source>
+        <translation>openpilot</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/util.cc" line="82"/>
+        <source>%n minute(s) ago</source>
```
Comment (sshane) [inline selfdrive/ui/translations/main_pt.ts:684]: Do you mind adding the plural forms for these?

Comment (AlexandreSato) [reply selfdrive/ui/translations/main_pt.ts:643]: Done!

File: selfdrive/ui/translations/main_pt.ts [hunk 2]
```diff
@@ -0,0 +1,1307 @@
...
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>Cancelar</translation>
+    </message>
+    <message numerus="yes">
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %n character(s)!</source>
+        <translation>
+            <numerusform>Necessita no mínimo %n caractere(s)!</numerusform>
+            <numerusform></numerusform>
```
Comment (sshane) [inline selfdrive/ui/translations/main_pt.ts:314]: I just noticed this, can we make sure all the numerus translations have plural forms?

**Comment by sshane** [comment]
I added a test that asserts all plural translations are completed now, if marked as finished

File: selfdrive/ui/translations/main_pt.ts (hunk 2 above)
Comment (AlexandreSato) [reply selfdrive/ui/translations/main_pt.ts]: Now I think I understand what was missing. I had thought i was right the first time but had not seen that a new field to be filled in the linguist had emerged.

**Comment by sshane** [comment]
Thanks for the PR!

**Pull request merged**